src/components/EconomieTotale.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div class="bg-white shadow-lg flex p-4 flex-col rounded-2xl w-3/4 h-full">
+    <div class="text-2xl text-black-600 font-bold">Economie Totale</div>
+    <div class="mt-2.5 w-full flex flex-row h-full">
+      <div class="flex flex-col w-full mr-2 bg-white shadow-md p-4 items-center justify-center text-left rounded-2xl">
+        <h3 class="w-full text-lg text-black-600 font-bold">Consommation d'énergie</h3>
+        <div class="mt-3 w-full text-black-600">
+          <span class="text-6xl font-bold">32%</span>
+          <span class="ml-2 text-base">d’énergie économisée</span>
+        </div>
+        <div class="mt-3 w-full">
+          <span class="text-xl text-green-400">+</span>
+          <span class="text-xl ml-2 text-green-400">3%</span>
+          <span class="text-sm ml-3 text-gray-400">par rapport à la semaine dernière</span>
+        </div>
+      </div>
+
+      <div class="flex flex-col w-full ml-2 bg-white shadow-md p-4 items-center justify-center text-left rounded-2xl">
+        <h3 class="w-full text-lg text-black-600 font-bold">Budgétaire</h3>
+        <div class="mt-3 w-full text-black-600">
+          <span class="text-6xl font-bold">1145€</span>
+          <span class="ml-2 text-base">de budget économisé</span>
+        </div>
+        <div class="mt-3 w-full">
+          <span class="text-xl text-green-400">+</span>
+          <span class="text-xl ml-2 text-green-400">5%</span>
+          <span class="text-sm ml-3 text-gray-400">par rapport à la semaine dernière</span>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
```

Comment:
Utiliser la syntaxe Vue3
```suggestion
export default defineComponent({})
```